sdk/communication/azure-communication-chat/ci.yml

```
@@ -0,0 +1,35 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+      - main
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+      - sdk/communication/azure-communication-chat/
+    exclude:
+      - sdk/communication/azure-communication-chat/pom.xml
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+      - sdk/communication/azure-communication-chat/
+    exclude:
+      - sdk/communication/azure-communication-chat/pom.xml
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
```

```suggestion
  template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
```
Similar for the others

---

Isn't this path relative to this file? If so how does '/eng/pipelines' work here?

---

I got my answer. It is relative to the root! Updating ...

---

Yes starting with / makes it absolute from the root.